spec/13-functions.md

```diff
@@ -74,6 +74,7 @@ A function is called via the function-call operator [`()`](10-expressions.md#fun
 
   <i>return-type:</i>
     : <i>type-declaration</i>
+    : void
```

Comment:
Also, the spec doesn't cover that the new PHP 7.0 and 7.1 types are a special kind of reserved word. I'll make a separate pull request for that.